addons/optionsmenu/XEH_preInit.sqf

```diff
@@ -15,4 +15,55 @@ GVAR(serverSideValues) = [];
 GVAR(categories) = [];
 GVAR(currentCategorySelection) = 0;
 
+
+[] spawn {
+    sleep 0.1; //is3den command has problems at pre-init, need spawn/sleep
```

Comment:
by "in 3den" he means while placing objects. postInit will fire once you press "preview", but not while you actually are in the editor.